### DISCUSSION TITLE
REST Api to install github app into multiple orgs

### DISCUSSION THREAD

**Discussion by amartyaroy** [community/community#103156] [Enterprise]
Hi All,
I have an enterprise account where I am having 40+ orgs are being created.Each orgs are being created by different customers and users.Now i am stuck in an issue where i have created one github app in an orgs lets say A and i want this app to be installed in rest of the 39 orgs. So if you go through UI click and install method its going to be very tough for us to install the app using this.I was thinking is there any REST api which can do this job for us.Where we can provide the app related information and then the api will be installed automatically in which ever target org we want. Currently we have api to install api in repo level but if we can get on org level then i guess this usecase can solve larger audience who have multiple orgs created in the enterprise level

**Comment by AnthonyHerman** [comment]
Kind of surprised this doesn't exist. Similar situation. Need an app installed into 500+ orgs.
